tests/default_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	_ "go-google-scraper-challenge/routers"
```

Comment:
Should we group the import following our [Go conventions](https://nimblehq.co/compass/development/code-conventions/golang/#imports)?